build.gradle

```diff
@@ -176,6 +176,10 @@ subprojects {
   javadoc.options {
     encoding = 'UTF-8'
     links 'https://docs.oracle.com/javase/7/docs/api/'
+
+    if (JavaVersion.current().isJava8Compatible()) {
+      addStringOption('Xdoclint:all,-missing', '-quiet')
+    }
```

Comment:
This sounds like a good thing to do regardless of version. Any reason this is gated to Java 8?

Reply:
doclint didn't exist before java8 :)